plugins/woocommerce/tests/api-core-tests/playwright.config.js

```diff
@@ -0,0 +1,50 @@
+const { devices } = require( '@playwright/test' );
+require( 'dotenv' ).config();
```

Comment:
We chatted on this a bit, but let's see if we can find a way to make this local to this folder as a start. Or, perhaps, we can look at hardcoding it for now as the Playwright e2e tests are doing--then look at making it variable in a future enhancement.